v2.0/partitioning.md

```diff
@@ -0,0 +1,371 @@
+~~~
+
+#### Step 2: Set the enterprise license
+
+To set the enterprise license, see [Set the Trial or Enterprise License Key](enterprise-licensing.html#set-the-trial-or-enterprise-license-key)
+
+#### Step 3: Create a table with the appropriate partitions:
+
+~~~ sql
+> CREATE TABLE students (
+    id SERIAL, 
+    name STRING,
+    email STRING,
+    country STRING, 
+    expected_graduation_date DATE, 
+    PRIMARY KEY (country, expected_graduation_date, id))
+    PARTITION BY LIST (country)(
+        PARTITION australia VALUES IN ('AU','NZ') PARTITION BY RANGE (expected_graduation_date)(PARTITION graduated_au VALUES FROM (MINVALUE) TO ('2017-08-15'), PARTITION current_au VALUES FROM ('2017-08-15') TO (MAXVALUE)),
+        PARTITION north_america VALUES IN ('US','CA') PARTITION BY RANGE (expected_graduation_date)(PARTITION graduated_us VALUES FROM (MINVALUE) TO ('2017-08-15'), PARTITION current_us VALUES FROM ('2017-08-15') TO (MAXVALUE))
+    );
+~~~
+
+Subpartition names must be unique within a table. In our example, even though `graduated` and `current` are sub-partitions of distinct partitions, they still need to be uniquely named. Hence the names `graduated_au`, `graduated_us`, and `current_au` and `current_us`.
+
+#### Step 4: Create corresponding zone configurations:
+
+~~~ shell
+$ cat > current_us.zone.yml
+constraints: [+ssd,+datacenter=us1]
+
+$ cat > graduated_us.zone.yml
+constraints: [+hdd,+datacenter=us1]
+
+$ cat > current_au.zone.yml
+constraints: [+ssd,+datacenter=au1]
+
+$ cat > graduated_au.zone.yml
+constraints: [+hdd,+datacenter=au1]
+~~~
+
+#### Step 5: Apply zone configurations to corresponding partitions:
+
+~~~ shell
+$ cockroach zone set roachlearn.students.current_us --insecure -f current_us.zone.yml
+$ cockroach zone set roachlearn.students.graduated_us --insecure -f graduated_us.zone.yml
+
+$ cockroach zone set roachlearn.students.current_au --insecure -f current_au.zone.yml
+$ cockroach zone set roachlearn.students.graduated_au --insecure -f graduated_au.zone.yml
+~~~
+
+### Repartition a Table
+
+Consider the partitioned table of students of RoachLearn. Suppose the table has been partitioned on range to store the current students on fast and expensive storage devices (example: SSD) and store the data of the graduated students on slower, cheaper storage devices(example: HDD). Now suppose we want to change the date after which the students will be considered current to `2018-08-15`. We can achieve this by using the `ALTER TABLE` command. 
+
+~~~ sql
+> ALTER TABLE students_by_range PARTITION BY RANGE (expected_graduation_date) (
+    PARTITION graduated VALUES FROM (MINVALUE) TO ('2018-08-15'), 
+    PARTITION current VALUES FROM ('2018-08-15') TO (MAXVALUE));
+~~~
+
+### Unpartition a Table
+
+You can remove the partitions on a table by using the `PARTITION BY NOTHING` syntax with the `ALTER TABLE` command:
+ 
+~~~ sql
+> ALTER TABLE students PARTITION BY NOTHING;
+~~~
+
+## Licensing Requirements
```

Comment:
Perhaps you should move this under **How Table Partitioning Works**. I mentioned above that it feels like a gap there in any case.